batch_job/src/work_queue_factory.c

```diff
@@ -1161,6 +1187,13 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	memset(cmd,'\0',1024*sizeof(char));
```

Comment:
bzero(cmd, sizeof(*cmd));

Reply:
Isn't bzero a gnu specific function in string.h?

Reply:
Yes, I did not know it was deprecated... I liked it had less confusing order of arguments.

Do remove the 1024. Instead use sizeof(cmd);